app/src/main/java/org/wikipedia/analytics/eventplatform/BreadCrumbLogEvent.kt

```diff
@@ -71,5 +71,9 @@ class BreadCrumbLogEvent(
             val str = "$viewReadableName." + (view as TextView).text
             EventPlatformClient.submit(BreadCrumbLogEvent(BreadCrumbViewUtil.getReadableScreenName(context), str))
         }
+
+        fun logMenuItemSelection(context: Context, actionStr: String) {
```

Comment:
Based on earlier comments, can this become `logMenuItemSelection(context: Context, item: MenuItem)`?

Reply:
Good idea @dbrant. Addressed it with a little alteration. Lmk if it looks good now.